content/concepts/projects-using-ocean.md

```diff
@@ -48,7 +48,7 @@ These are live projects that leverage core functionality of Ocean, such as readi
 
 ## Using OCEAN
 
-These projects use just the Ocean Token, but do more than just exchanging it. This includes staking, and liquidity mining opportunities.
+These projects use just OCEAN token, but do more than just exchanging it. This includes staking, and liquidity mining opportunities.
```

Comment:
I think we had this standard of using either the ERC-20 name, or the symbol but not sure we stayed that consistent across docs:

- "Ocean Token"
- "OCEAN"

Reply:
Good eye, I made it all consistently to "OCEAN" here.